corehq/apps/ota/views.py

```diff
@@ -199,7 +199,7 @@ def get_restore_params(request):
         openrosa_headers = getattr(request, 'openrosa_headers', {})
         openrosa_version = openrosa_headers[OPENROSA_VERSION_HEADER]
     except KeyError:
-        openrosa_version = request.GET.get('openrosa_version', OPENROSA_DEFAULT_VERSION)
+        openrosa_version = request.GET.get('openrosa_version', None)
```

Comment:
@proteusvacuum you'd probably be interested in this commit.  I needed openrosa verson for something else (see the rest of the PR if interested), so I saved it as-is to the `RestoreParams` and dropped the business of defaulting to `1.0`.  Now that fallback behavior is defined in the `async_restore_enabled` calculation.  The feature I'm doing wants the opposite approach, so saving it as `None` when not specified will allow (force?) each use case to decide how to handle unspecified versions.